manifests/w/wherewhere/WSATools/0.0.1/wherewhere.WSATools.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: wherewhere.WSATools
+PackageVersion: 0.0.1
+Installers:
+- InstallerLocale: zh-CN
+  Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://github.com/Paving-Base/WSA-Tools/releases/download/v0.0.1/WSATools.Installer_0.0.1.0_x64.msi
+  InstallerSha256: 618A1967B5993CA12FDC4693CD5D227AFB5F721FCE580E741EC4320EED334F73
+  ProductCode: '{9FC7AB78-1DD2-4E2D-8151-093D249DA9D2}'
+- InstallerLocale: zh-CN
+  Architecture: x86
+  InstallerType: msi
+  InstallerUrl: https://github.com/Paving-Base/WSA-Tools/releases/download/v0.0.1/WSATools.Installer_0.0.1.0_x86.msi
+  InstallerSha256: 273DD0027C1E184F757D6B95DDA5642B186DE295D4229811FB8E57643CA072A0
+  ProductCode: '{9FC7AB78-1DD2-4E2D-8151-093D249DA9D2}'

```

Comment:
```suggestion
```
Looks like the x86 installer is corrupted.

This is the error I receive:
![image](https://user-images.githubusercontent.com/11600822/141688790-0f3ac211-d314-4523-aecd-70b83459f7ff.png)